projects/engine/src/plugin/external.ts

```diff
@@ -31,6 +35,11 @@ export abstract class ExternalPlugin extends Plugin {
     super.deactivate()
   }
 
+  /** Set options for an external plugin */
+  setOptions(options: Partial<ExternalPluginOptions> = {}) {
```

Comment:
is this needed to overrude this? `setOptions` will anyway be available

Reply:
For the type in the params